google/oauth2/flow.py

```diff
@@ -54,12 +54,22 @@
 """
 
 import json
+import logging
+import webbrowser
+
+from six.moves import BaseHTTPServer
+from six.moves import http_client
+from six.moves import input
+from six.moves import urllib
 
 import google.auth.transport.requests
 import google.oauth2.credentials
 import google.oauth2.oauthlib
 
 
+_LOGGER = logging.getLogger(__name__)
```

Comment:
This is so the the request handler below doesn't write to stderr, but instead uses our named logger.